browser-test/src/admin/admin_application_statuses.test.ts

```diff
@@ -359,6 +361,171 @@ test.describe('view program statuses', () => {
+        }
+        expect(await notifyCheckbox.isChecked()).toBe(true)
+
+        expect(await modal.innerText()).toContain(
+          ' of this change at ' + guestEmail,
+        )
+        await adminPrograms.confirmStatusUpdateModal(modal)
+        expect(await adminPrograms.getStatusOption()).toBe(emailStatusName)
+        await adminPrograms.expectUpdateStatusToast()
+
+        if (supportsEmailInspection()) {
+          await adminPrograms.expectEmailSent(
+            emailsBefore.length,
+            guestEmail,
+            emailBody,
+            programWithStatusesName,
+          )
+        }
+      })
+      test('both email addresses are displayed and two emails are sent for a logged in user that has answered the PAI email question with a different email', async () => {
+        const {page, applicantQuestions, adminPrograms} = ctx
+        const otherTestUserEmail = 'other@example.com'
+
+        await loginAsTestUser(page)
+        await applicantQuestions.applyProgram(programWithStatusesName)
+        await applicantQuestions.answerEmailQuestion(otherTestUserEmail)
+        await applicantQuestions.clickNext()
+        await applicantQuestions.submitFromReviewPage()
+        await logout(page)
+
+        await loginAsProgramAdmin(page)
+        await adminPrograms.viewApplications(programWithStatusesName)
+        await adminPrograms.viewApplicationForApplicant(testUserDisplayName())
+
+        const acccountEmailsBefore = supportsEmailInspection()
+          ? await extractEmailsForRecipient(page, testUserDisplayName())
+          : []
+        const applicantEmailsBefore = supportsEmailInspection()
+          ? await extractEmailsForRecipient(page, otherTestUserEmail)
+          : []
+
+        const modal =
+          await adminPrograms.setStatusOptionAndAwaitModal(emailStatusName)
+        expect(await modal.innerText()).toContain(
+          ' of this change at ' +
+            testUserDisplayName() +
+            ' and ' +
+            otherTestUserEmail,
+        )
+
+        await adminPrograms.confirmStatusUpdateModal(modal)
+        await adminPrograms.expectUpdateStatusToast()
+
+        if (supportsEmailInspection()) {
+          await adminPrograms.expectEmailSent(
+            acccountEmailsBefore.length,
+            testUserDisplayName(),
+            emailBody,
+            programWithStatusesName,
+          )
+          await adminPrograms.expectEmailSent(
+            applicantEmailsBefore.length,
+            otherTestUserEmail,
+            emailBody,
+            programWithStatusesName,
+          )
+        }
+      })
+      test('only one email is displayed and sent for a logged in user that has answered the PAI email question with the same email they used to login', async () => {
+        const {page, applicantQuestions, adminPrograms} = ctx
+
+        await loginAsTestUser(page)
+        await applicantQuestions.applyProgram(programWithStatusesName)
```

Comment:
This test is consistently flaky when I run in locally and times out trying to find `continue-application-button`. Any idea what's up with that?
![Untitled](https://prod-files-secure.s3.us-west-2.amazonaws.com/bfe8b4f1-d20b-47cf-ad3a-c68c7354f5e1/581015dc-51d6-4dd0-83ef-73e798268aa3/Untitled.png)

Reply:
When this passes, how long did the retry take? It could be there's something that sometimes takes longer to run. If you think it's slow you can modify this test to be `test.slow('only one email....blah blah`. Tests marked as slow get three times the amount of timeout they usually get. That's where I'd start first.

Reply:
Great callout, I'll see if that's what's happening.

Reply:
It's failing because I'm logging is as the same test user in both tests, and since the data isn't cleared between tests, I need to click "Edit" instead of "Continue" to edit the previously submitted value. It's surprising to me that it's passing on retry and in CI. I wonder if the data shared between tests is handled differently in those environments? I'm not sure how to get around this.

Reply:
I can update the test to click the correct buttons, but then there's an issue with needing to create a new status since one was already set for this user which means I need to update a bunch of snapshots.

Reply:
There were too many errors from test context carrying over so I ended up factoring this out into it's own `describe` that clears the db between tests.